build.gradle.kts

```diff
@@ -24,4 +45,9 @@ dependencies {
   runtimeOnly("com.h2database:h2:1.4.200")
 
   implementation("io.springfox:springfox-boot-starter:3.0.0")
+  testImplementation("au.com.dius.pact.provider:junit5spring:4.2.2")
```

Comment:
Hi John - this mix of libraries was a mistake from me. For provider I think you need to lose thet 4.0.10 . I think for the provider side we just need this
```
    testImplementation("au.com.dius.pact.provider:junit5spring:$pactVersion")
    testImplementation("au.com.dius.pact.provider:junit5:$pactVersion")
```

Reply:
So I've had a look at this and updated it to the minimal set of dependencies that seems to work... `testImplementation("au.com.dius.pact.provider:junit5:$pactVersion")` seems to be redundant with `testImplementation("au.com.dius.pact.provider:junit5spring:$pactVersion")` and the jvm dependency is the latest available and it won't build without 🤔  If you don't think that's right probably best to jump on a call to sort it